packages/react-native/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@aws-amplify/ui-react-native",
```

Comment:
I'd put `private: "true"` here, just as safety measures until we're ready to publish this to NPM.

Reply:
nvm, it's at line 8 :p